### ISSUE TITLE
FutureWarning when using Layer.create

### ISSUE THREAD

**Issue by bgraedel** [napari/napari#4344]
## 🐛 Bug

Numpy raises a future warning when using Layer.create
## To Reproduce

```
import numpy as np
import napari
from napari.layers import Layer

viewer = napari.Viewer()
data = (
np.random.random((10, 2)) * 20,
 {'face_color': 'blue'},
  'points')

Layer.create(*data)
```
shows:
```
FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  return bool(asarray(a1 == a2).all())
```
## Expected behavior

A points layer is added (which also is added but with the above mentioned warning shown)
## Environment

napari: 0.4.15
Platform: Windows-10-10.0.22000-SP0
Python: 3.10.4 | packaged by conda-forge | (main, Mar 24 2022, 17:32:50) [MSC v.1929 64 bit (AMD64)]
Qt: 5.15.2
PyQt5: 5.15.6
NumPy: 1.22.3
SciPy: 1.8.0
Dask: 2022.03.0
VisPy: 0.9.6

OpenGL:
- GL version: 4.6.0 - Build 30.0.101.1003
- MAX_TEXTURE_SIZE: 16384

Screens:
- screen 1: resolution 2560x1440, scale 1.0
- screen 2: resolution 1280x720, scale 3.0

Plugins:
- console: 0.0.4
- scikit-image: 0.4.15
- svg: 0.1.6

**Comment by cnstt** [comment]
I have the same issue, when adding a Points layer I get the following warning:

`/usr/lib64/python3.10/site-packages/numpy/core/numeric.py:2446: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison!
`

**Comment by cnstt** [comment]
Might this be something related to array_equal() of numpy, maybe in the tests here:
https://github.com/napari/napari/blob/1b3d9521351be7be4905da7419a9179cc99df0ca/napari/layers/utils/_tests/test_layer_utils.py
ping @brisvag (sorry for pinging you, I don't really know who to ask, I might try to debug it myself, but have no experience in napari development/nor have a development environment setup for napari development yet)

**Comment by brisvag** [comment]
No worries :) Yeah this issue has been aroudn some time, I also saw that warning, I guess no one was annoyed enough to make it their priority :P I'm not sure what triggers it (surely not the tests, since those are not executed at runtime). Maybe @jni is more familiar with numpy deprecation stuff?

**Comment by andy-sweet** [comment]
@bgraedel : I believe this was fixed by https://github.com/napari/napari/pull/4256, which was included in v0.4.16, so I'm going to close this.

If anyone still finds this issue, please reopen and comment.
